client/my-sites/plugins/plugin-list-header/index.jsx

```diff
@@ -309,7 +309,7 @@ export default React.createClass( {
 			<SelectDropdown compact
 					className="plugin-list-header__actions_dropdown"
 					key="plugin-list-header__actions_dropdown"
-					selectedText={ this.translate( 'Actions' ) } >
+					selectedText={ this.translate( 'Actions', { context: 'context added' }) } >
 
 				<DropdownItem key="plugin__actions_title" selected={ true } value="Actions">
 					{ this.translate( 'Actions' ) }
```

Comment:
Alternate string suggestion: ```translate( 'Update %(numUpdates)d Plugin ', 'Update %(numUpdates)d Plugins' )```[Details](http://translate.wordpress.com)Alternate string suggestion: ```translate( 'Update %(numUpdates)d Plugin ', 'Update %(numUpdates)d Plugins' )```[Details](http://translate.wordpress.com)Alternate string suggestion: ```translate( 'Update %(numUpdates)d Plugin ', 'Update %(numUpdates)d Plugins' )```[Details](http://translate.wordpress.com)Alternate string suggestion: ```translate( 'Update %(numUpdates)d Plugin ', 'Update %(numUpdates)d Plugins' )```[Details](http://translate.wordpress.com)Warning: 3 translations will be lost with this change.
[Details](http://translate.wordpress.com)Warning: 3 translations will be lost with this change.
[Details](http://translate.wordpress.com)Warning: 3 translations will be lost with this change.
[Details](http://translate.wordpress.com)Warning: 3 translations will be lost with this change.
[Details](http://translate.wordpress.com)Warning: 3 translations will be lost with this change.
[Details](http://translate.wordpress.com)Warning: 3 translations will be lost with this change.
[Details](http://translate.wordpress.com)Warning: 3 translations will be lost with this change.
[Details](http://translate.wordpress.com)Warning: 3 translations will be lost with this change.
[Details](http://translate.wordpress.com)Warning: 3 translations will be lost with this change.
[Details](http://translate.wordpress.com)Warning: 3 translations will be lost with this change.
[Details](http://translate.wordpress.com)Warning: 3 translations will be lost with this change.
[Details](http://translate.wordpress.com)Warning: 3 translations will be lost with this change.
[Details](http://translate.wordpress.com)Warning: 3 translations will be lost with this change.
[Details](http://translate.wordpress.com)Warning: 3 translations will be lost with this change.
[Details](http://translate.wordpress.com)Warning: 3 translations will be lost with this change.
[Details](http://translate.wordpress.com)